guides/adding-commit-info/adding-commit-info.adoc

```diff
@@ -56,9 +56,43 @@ plugins {
 }
 ----
 
+:exclude-for-build:
+
+:exclude-for-build:gradle
+
+== Maven Git Commit Id Plugin
+
+If a `git.properties` file is available on the classpath, the http://docs.micronaut.io/latest/api/io/micronaut/management/endpoint/info/source/GitInfoSource.html[GitInfoSource] will expose the values in that file under the `git` key. Generation of a `git.properties` file will need to be configured as part of your build; for example, you may choose to use the https://github.com/git-commit-id/git-commit-id-maven-plugin[Maven Git Commit Id Plugin] plugin. The plugin provides a goal named `git-commit-id:revision` responsible for the `git.properties` file generation. It is automatically invoked upon the execution of the `classes` goal. You can find the generated file in the directory `target/classes`.
```

Comment:
Instead of repeating this paragraph: I will move it outside of the build specific section. 

```
If a `git.properties` file is available on the classpath, the http://docs.micronaut.io/latest/api/io/micronaut/management/endpoint/info/source/GitInfoSource.html[GitInfoSource] will expose the values in that file under the `git` key. Generation of a `git.properties` file will need to be configured as part of your build;
```